.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 # IMPORTANT: To update Docker image version, please search and update ":{previous_version}"
```

Comment:
either you're wrong or this comment is?

Reply:
i.e. I don't see any modifications of a `previous_version` in this diff, and the comment seems to say that you need one

Reply:
Err, the change is right below. previous_version here is 253.

Reply:
it says 'update previous_version and ALSO the version number below' - are they not two separate things?

Reply:
Version number below is https://github.com/pytorch/pytorch/pull/13231/files#diff-1d37e48f9ceff6d8030570cd36286a61R3

Update previous version is all occurrences of the number in the yaml: e.g., https://github.com/pytorch/pytorch/pull/13231/files#diff-1d37e48f9ceff6d8030570cd36286a61R429